pkgs/tools/graphics/xcur2png/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, pkgconfig, libpng, xorg }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "xcur2png";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "eworm-de";
+    repo = pname;
+    rev = version;
+    sha256 = "0858wn2p14bxpv9lvaz2bz1rk6zk0g8zgxf8iy595m8fqv4q2fya";
+  };
+
+  nativeBuildInputs = [
+    pkgconfig
+  ];
+
+  buildInputs = [
+    libpng
+    xorg.libXcursor
+    xorg.libXrender
```

Comment:
Where was the need for xrender specifed?

Reply:
`libXcursor` depends on `libXrender` and `xcur2png` depends on `libXcursor`. Without explicitly listing `libXrender` in `buildInputs` here, building `xcur2png` fails with the message:

```
checking for XCUR2PNG... configure: error: Package requirements (libpng >= 1.0.6 xcursor) were not met:

Package 'xproto', required by 'xcursor', not found
```

(In [VoidLinux](https://github.com/void-linux/void-packages/blob/master/srcpkgs/xcur2png/template) it also has this dependency.)

Reply:
Ahh, `libXrender` propagates 
```
xorg.xorgproto
xorg.libX11
```

If you add those and drop `libXrender` I think we should be good.

Reply:
Done. Thanks for pointing that out.